lib/ocpp/v201/device_model.cpp

```diff
@@ -236,6 +237,15 @@ GetVariableStatusEnum DeviceModel::request_value_internal(const Component& compo
         return GetVariableStatusEnum::UnknownVariable;
     }
 
+    if (variable_it->second.ocpp_version.has_value() && variable_it->second.ocpp_version.value() != ocpp_version) {
+        EVLOG_warning << "Trying to get device model value " << variable_id.name.get()
+                      << ", but this variable is not "
+                         "for this ocpp version, but for ocpp version "
+                      << ::ocpp::conversions::ocpp_protocol_version_to_string(variable_it->second.ocpp_version.value());
```

Comment:
something like this:
```suggestion
        EVLOG_warning << "Trying to get device model value " << variable_id.name.get()
                      << ", but this variable is not available in OCPP version " << ocpp_version
                      << " just for OCPP version "
                      << ::ocpp::conversions::ocpp_protocol_version_to_string(variable_it->second.ocpp_version.value());
```